### DISCUSSION TITLE
Please help test & translate upcoming 3.4

### DISCUSSION THREAD

**Discussion by kjk** [sumatrapdfreader/sumatrapdf#2583] [Announcements]
I'm hoping to release 3.4 in a week or so. 

Now is a good time to do some extra testing on [latest pre-release](https://www.sumatrapdfreader.org/docs/Contribute-to-SumatraPDF) and [update translations](https://www.apptranslator.org/app/SumatraPDF) for your language.

So please download [latest pre-release](https://www.sumatrapdfreader.org/docs/Contribute-to-SumatraPDF) and try things and file bugs if you encounter some.

**Comment by X1aSheng** [comment]
ok

**Comment by pritkr** [comment]
length of individual tabs in tab bar is too big

**Reply by GitHubRulesOK** [reply]
TabWidth = 40
![image](https://user-images.githubusercontent.com/12754913/169310057-d3fa08c3-b586-473e-a34a-aa40723cabb0.png)

**Reply by kjk** [reply]
You can use advanced options to customize it with `TabWidth` setting. See https://www.sumatrapdfreader.org/settings/settings

**Reply by pritkr** [reply]
they are a little longer in default width

**Comment by ZYinMD** [comment]
Hi, can you provide a list of new features? There are some features I'm looking forward to, but not sure if it's included in 3.4.

**Comment by kjk** [comment]
See https://www.sumatrapdfreader.org/docs/Version-history, expand **>Next** section

**Reply by ZYinMD** [reply]
Thanks! I've actually come to this page before, but somehow didn't pay attention to the "Next" button. Maybe you could change it to "Next version". A `cursor: pointer` doesn't hurt either.

**Comment by ZYinMD** [comment]
I tested a custom keyboard shortcut with `CmdEditAnnotations`. Here's some issues about it:

1. Previously, I used to right click on a highlight, then choose "Edit annotations", which will open the annotation window, then focus on the highlight that I clicked on, which is good. However, the `CmdEditAnnotations` hotkey will only open the annotation window, but not focus on any item. 

   Desired behavior: if I click on a highlight then press the hotkey, the annotation window shows, and the highlight is focused. You could choose to implement it with either left click or right click.

   Current behavior: neither left click nor right click followed by the hotkey will help me focus on the item clicked.


1. In full screen mode, if I press the hotkey, the annotation window will show, but the rendering is buggy. If this is hard to fix, you could make it exit the full screen mode first, then show the annotation window.

1. Related to **2**: in v3.3, if you right click in full screen mode, the context menu won't include annotation related items. I think it's better to include those items, but exit the full screen mode on clicking them, than not include them.

**Comment by GitHubRulesOK** [comment]
@kjk 
retrospective comment after the fact the exe copyright date is 2021 :-)
seems from https://github.com/sumatrapdfreader/sumatrapdf/blob/ab79e929f6ceb48f92a7775597496374c8cbad01/src/Version.h#L53
